nss-ivory-trolls/BangazonSite
# Jb tues ticket8

## Ticket #1 
Note this is for ticket 1 despite branch name indicating it's for ticket8... while messin with 8 I found some stuff that needed to be sorted for 1

Given a user is viewing any page on the Bangazon site
When the user performs a gesture on the Product Categories hyperlink
Then the user will see a view containing a list of all product categories
And next to the category name, the number of products in that category will be displayed
And the first three products in the category will be displayed beneath each category, that are hyperlinks to the product detail


Please delete options that are not relevant.



# Testing Instructions for Change Made

1.  `git fetch --all`
1.  `git checkout jb-tuesTicket8`
1.  run Bangazon
1.  in nav bar click "Products"  (don't use dropdown)
1.  you will see product types displayed along with amount of that particular product type available
1.  if particular product type doesn't have product in it, you will not see that product type listed
1.  product types will list no more then 3 of it's available products
1.  now click an available product under any of the Product Types (like "kite")
1.  you will now see a detailed description of the kite (Note: Add to order button does not work)
1.  in detailed description "back to list" hyperlink will return you to product type list you came from


# Checklist:
